Create Deploy homepage in docs (#3819)
PostHog/posthog.com: contents/docs/deploy.mdx

---
title: Deploy PostHog
sidebar: Docs
---

PostHog offers a number of different deployment options:

-   [PostHog Cloud](#posthog-cloud) - Create a new PostHog Cloud deployment with just a few clicks.
    -   Recommended as it's **the fastest way** to try PostHog out.
    -   Comes with **extra features for reliability and security** and automatic updates.
-   [Hobby](#hobby-deployment) - Recommended for testing or very small hobby projects _(&lt;100k events/month)_

> -   [PostHog Self-hosted](#posthog-self-hosted) - Recommended for most self-host deployments.

Reviewer: Self-hosted is recommended for people who need to self-host doesn't really add much. I'd suggest changing it to...

```suggestion
-   [PostHog Self-hosted](#posthog-self-hosted) - Recommended for users who need to keep data on their own infrastructure or private cloud.
```

## PostHog Cloud

> [PostHog Cloud](/pricing) is our managed hosting service and let's you start using PostHog in only a few clicks.

Reviewer: ```suggestion
[PostHog Cloud](/pricing) is our managed hosting service and enables you to start using PostHog in only a few clicks.
```

We generally recommend this option for most users as it's the easiest to try out and offers the most features out of the box:

-   **1 million events included _free_ every month!**
-   Highly-scalable infrastructure to handle large event volumes _(>100m events/month)_
-   Automatic updates when new features are released
-   Unlimited teammates and SSO/SAML _(Enterprise only)_

> 

Reviewer: ```suggestion

Change your mind later? You can always migrate to a self-hosted instance later.
```

Seems like it's worth throwing in a mention of this.

> <div class="flex flex-col items-stretch lg:flex-row lg:items-center justify-between bg-gray-accent-light py-4 px-6 rounded-sm mb-4 space-y-3 lg:space-y-0 lg:space-x-3">

Reviewer: <img width="682" alt="Screenshot 2022-07-21 at 11 24 05" src="https://user-images.githubusercontent.com/84011561/180192053-7854fc57-9d93-4881-9872-743cf5963221.png">

This doesn't display well in the Docs' dark mode option.

    <div class="space-y-1">
        <span class="font-semibold text-lg block">Get started with PostHog Cloud.</span>
        <span class="text-base text-gray block">The easiest and quickest way to set-up PostHog.</span>
    </div>
    <CallToAction class="shrink-0" to="/signup">Deploy now</CallToAction>
</div>

For more information, check out our [pricing](/pricing) page.

## Hobby deployment

The [Hobby](/docs/self-host/hobby) deployment is the most basic method for hosting PostHog yourself, and is suitable for testing or very small projects.
This deployment runs the entire PostHog stack on a single VM, meaning it can only handle relatively low event volumes (&lt;100k events/month).

> For more information on setting up a hobby deployment, check out [here](/docs/self-host/hobby) for instructions.

Reviewer: ```suggestion
For more information on setting up a hobby deployment, check out [our Hobby deployment docs](/docs/self-host/hobby) for instructions.
```

Generally best SEO-wise to make link text descriptive and avoid putting it on the word 'here'.

## PostHog Self-hosted

[PostHog Self-hosted](/docs/self-host) is the recommended method for hosting PostHog yourself and gives much better performance than the Hobby deployment.

This method offers a number of benefits over PostHog Cloud:

-   Customer data never leaves your infrastructure
-   First party cookies on your own domain
-   Full SQL access to your data _(Enterprise only)_

> However, this also means that you will need to manage all updates, scaling, and security by yourself.

Reviewer: ```suggestion
However, this also means that you will need to manage all updates, scaling, and security by yourself, or engage a partner from [the PostHog Marketplace](/marketplace) to assist you. 
```

Feel free to remove this if you disagree, just feels worth adding another option.

---
title: Deploy PostHog
sidebar: Docs
---

PostHog offers a number of different deployment options:

-   [PostHog Cloud](#posthog-cloud) - Create a new PostHog Cloud deployment with just a few clicks.

>     -   Recommended as it's **the fastest way** to try PostHog out.

Reviewer 2: ```suggestion
    -   Recommended as it's **the fastest way** to try PostHog out. It's also totally **free* for up to 1m events/month.
```

>     -   Comes with **extra features for reliability and security** and automatic updates.

Reviewer 2: > reliability and security

What are these? (maybe a link or remove?)

Author: Agreed, slightly tweaked this bullet-point

> -   [Hobby](#hobby-deployment) - Recommended for testing or very small hobby projects _(&lt;100k events/month)_

Reviewer 2: I'd swap these two (I'd put self hosted ahead of hobby) since self hosted is the option in line with ideal customer profile

-   [PostHog Self-hosted](#posthog-self-hosted) - Recommended for users who need to keep data on their own infrastructure or private cloud.

## PostHog Cloud

[PostHog Cloud](/pricing) is our managed hosting service and enables you to start using PostHog in only a few clicks.
We generally recommend this option for most users as it's the easiest to try out and offers the most features out of the box:

-   **1 million events included _free_ every month!**
-   Highly-scalable infrastructure to handle large event volumes
-   Automatic updates when new features are released

For more information, check out our [pricing](/pricing) page.

<div class="flex flex-col items-stretch lg:flex-row lg:items-center justify-between bg-gray-accent-light dark:bg-gray-accent-dark py-4 px-6 rounded-sm mb-4 space-y-3 lg:space-y-0 lg:space-x-3">
    <div class="space-y-1">
        <span class="font-semibold text-lg block">Get started with PostHog Cloud.</span>
        <span class="text-base text-gray block">The easiest and quickest way to set-up PostHog.</span>
    </div>

>     <CallToAction class="shrink-0" to="/signup">Deploy now</CallToAction>

Reviewer 2: ```suggestion
    <CallToAction class="shrink-0" to="/signup">Get started</CallToAction>
```
